lib/test.js

```diff
@@ -0,0 +1,14 @@
+const spawn = require('cross-spawn')
+
+module.exports = ({
+  inputDir,
+  testOpts = ['--opts', `${__dirname}/../mocha/mocha.opts`],
```

Comment:
Maybe I'm misunderstanding and you can make this a bit more clear. This `testOpts` thing is purely for setting the opts location for mocha, if you want to override it with your own. Other arguments come through via the `extras` arguments.

e.g.

You might use `brb test -- -w` to get it watching